README.md

```diff
@@ -2,11 +2,50 @@
 
 ## Stage 1
 
-This is a [proposal](https://tc39.github.io/process-document/) to introduce a separator in numeric literals in [ECMAScript](https://github.com/tc39/ecma262/).
+This is a [proposal](https://tc39.github.io/process-document/), the result of a merge between an earlier draft of itself and Christophe Porteneuve's [proposal-numeric-underscores](https://github.com/tdd/proposal-numeric-underscores), extends the existing [_NumericLiteral_](https://tc39.github.io/ecma262/#prod-NumericLiteral) to allow a separator character between digits.
+
 
 ## Motivation
 
-This feature enables developers to make their numeric literals more readable by creating a visual separation between groups of digits. 
+This feature enables developers to make their numeric literals more readable by creating a visual separation between groups of digits. Large numeric literals are difficult for the human eye to parse quickly, especially when there are long digit repetitions. This impairs both the ability to get the correct value / order of magnitude...
```

Comment:
There are a lot of examples now (also in the examples section). Couple of questions:

(a) do you think this is the most representative example of all to be in the motivation section?
(b) are there too many examples now?

I think the answer for (b) is "no, but maybe", and (a) is a bit subjective so we'll just have to pick our favorite (I just want to make sure it is deliberate, in that most people are going to get hung up on the motivation section before moving on).

Any favorites for (a)?